chemprop/v2/tests/unit/data/test_dataset.py

```diff
@@ -83,25 +83,25 @@ def test_smis(dataset, smis):
 
 
 def test_targets(dataset, targets):
-    np.testing.assert_array_equal(dataset.targets, targets)
+    np.testing.assert_array_equal(dataset.Y, targets)
 
 
 def test_set_targets_too_short(dataset):
     with pytest.raises(ValueError):
-        dataset.targets = np.random.rand(len(dataset) // 2, 1)
+        dataset.Y = np.random.rand(len(dataset) // 2, 1)
 
 
 def test_num_tasks(dataset, targets):
-    assert dataset.num_tasks == targets.shape[1]
+    assert dataset.t == targets.shape[1]
 
 
 def test_aux_nones(dataset: MoleculeDataset):
-    assert dataset.X_f is None
-    assert dataset.X_phase is None
-    assert dataset.V_fs is None
-    assert dataset.E_fs is None
-    assert dataset.gt_mask is None
-    assert dataset.lt_mask is None
-    assert dataset.d_v is None
+    assert (dataset.X_f == None).all()
+    # assert dataset.X_phase is None # there's no attribute in dataset that is relevant to phase?
+    assert (dataset.V_fs == None).all()
+    assert (dataset.E_fs == None).all()
+    assert (dataset.gt_mask == None).all()
+    assert (dataset.lt_mask == None).all()
```

Comment:
not sure why the API needed to change for this?

Reply:
`dataset.V_fs`, `dataset.E_fs`, `dataset.gt_mask`, and `dataset.lt_mask` are numpy arrays with `None` as elements, so I test that all the elements are `None`